configure.ac

```diff
@@ -976,6 +976,18 @@ AC_SUBST(JEMALLOC_LDFLAGS)
 AC_SUBST(JEMALLOC_AUTOCONF_FLAGS)
 fi
 
+
+AC_ARG_WITH(crash_privacy,       [  --with-crash_privacy=yes,no         If you want your crashes to not include names of symbols present in the binary. ], [], [with_crash_privacy=no])
```

Comment:
The option is confusing does `no` mean no privacy?
How does this work together, can I do `--with-crash_privacy=no --with-structured_crashes=no` ?

Reply:
The two options are independent, yeah. And I made the default be to "no", but I can toggle this from a "with" to a "disable" if you want mono to compile to be in "telemetry mode" by default instead. 

Would you have named this something else?

Reply:
Which use case is there for e.g. `--with-crash_privacy=yes --with-structured_crashes=no` ? Do we print the info it hides anywhere else or just in the json crash dump?

We could just merge this into the `--with-structured_crashes` option: `yes,hide_symbol_names,no`.

Reply:
Structured crashes are not merp crashes. Structured crashes are json dumps made by mono during crashes. Merp upload is going to use this code is a very specific way, and is enabled at runtime with an icall.